service/controller/config.go

```diff
@@ -0,0 +1,115 @@
+package controller
```

Comment:
All below is some copy+paste + some adjustments with stub `configuration` resource.